core/src/main/java/org/eclipse/hono/util/MessageHelper.java

```diff
@@ -104,6 +104,27 @@
      */
     public static final String APP_PROPERTY_TENANT_ID = "tenant_id";
 
+    /**
```

Comment:
Should we add somewhere in the docs what properties can be set as defaults?

Reply:
I tried to do this in the protocol adapter user guides, e.g. https://www.eclipse.org/hono/user-guide/http-adapter/#downstream-meta-data ...

Reply:
OK, now I see it. So only ttl and content-type can be set.

Reply:
for the time being: yes. For most of the AMQP defined properties, a default value doesn't make much sense, or are there others you would like to support?

Reply:
No, it's all good. I just wanted to understand it.